packages/core/src/Toggle/Toggle.stories.tsx

```diff
@@ -1,30 +1,276 @@
+  IToggleProps,
+  Image,
+  Label,
+  Radio,
+  Text,
+  ThemeProvider,
+  Toggle,
+} from '..'
 
-export default {
-  title: 'Toggle',
-  component: Toggle,
+import alignmentImage from './Toggle.Image.Alignment.png'
+import heroImage from './Toggle.Image.Hero.png'
+import statesImage from './Toggle.Image.States.png'
+import usageImage from './Toggle.Image.Usage.png'
+
+type ToggleStory = StoryObj<IToggleProps>
+
+export const Playground: ToggleStory = {
+  render: (args) => {
+    const [isOn, setIsOn] = useState(false)
+    const onToggle = () => setIsOn(!isOn)
+
+    return <Toggle isOn={isOn} onToggle={onToggle} {...args} />
+  },
+  argTypes: {
+    isOn: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+      defaultValue: false,
+    },
+  },
 }
 
-export const ToggleComponent = () => {
-  const [isOn, setIsOn] = useState(false)
-  const onToggle = () => setIsOn(!isOn)
-
-  const DollarIcon = (props) => <Dollar color={isOn ? 'primary.base' : 'background.dark'} {...props} />
-  const icon = <DollarIcon size={22} />
-  const largeIcon = <DollarIcon size={48} />
-
-  return (
-    <>
-      <h4>Default</h4>
-      <Toggle label='Default' isOn={isOn} onToggle={onToggle} />
-      <h4>Large</h4>
-      <Toggle label='Large' isOn={isOn} onToggle={onToggle} height={50} />
-      <h4>With Icon</h4>
-      <Toggle label='With Icon' isOn={isOn} onToggle={onToggle} icon={icon} />
-      <h4>Large With Icon</h4>
-      <Toggle label='Large With Icon' isOn={isOn} onToggle={onToggle} height={50} icon={largeIcon} />
-    </>
-  )
+export const Sizes: ToggleStory = {
+  render: () => {
+    const [isOn, setIsOn] = useState(false)
+    const onToggle = () => setIsOn(!isOn)
+
+    return (
+      <StoryStage display='flex'>
+        <Toggle isOn={isOn} onToggle={onToggle} />
+        <Toggle isOn={isOn} onToggle={onToggle} height={16} />
+      </StoryStage>
+    )
+  },
+  argTypes: {
+    isOn: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+      defaultValue: false,
+    },
+  },
+}
+
+export const WithIcon: ToggleStory = {
+  render: () => {
+    const [isOn, setIsOn] = useState(false)
+    const onToggle = () => setIsOn(!isOn)
+    const Icon = isOn ? CheckIcon : CloseIcon
+
+    return <Toggle isOn={isOn} onToggle={onToggle} icon={<Icon p={1} />} />
+  },
+  argTypes: {
+    isOn: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+      defaultValue: false,
+    },
+  },
+}
+
+const meta: Meta<typeof Toggle> = {
+  title: 'Actions/Toggle',
+  component: Toggle,
+  parameters: {
+    docs: {
+      page: () => (
+        <ThemeProvider>
+          <Hero name='Toggle' img={heroImage}>
+            Buttons allow users to trigger an action or event with a single click. For example, you can use a
+            button for allowing the functionality of submitting a form, opening a dialog, canceling an action,
+            or performing a delete operation.
+          </Hero>
```

Comment:
This doesn't seem related to `Toggle`. Is that intentional?

Reply:
This was due to copying the Button's docs as a template, thanks for catching this (it seems pretty obvious in hindsight 🤦 )